client/js/components/main/assessment.jsx

```diff
@@ -4,23 +4,18 @@ import React                                  from "react";
 import { connect }                            from "react-redux";
 
 import * as AssessmentActions                 from "../../actions/assessment";
+import * as AssessmentProgress                from "../../actions/assessment_progress";
 import appHistory                             from "../../history";
 import Item                                   from "../assessments/item";
 import Loading                                from "../assessments/loading";
 import ProgressDropdown                       from "../common/progress_dropdown";
-import { questionCount, questions, outcomes } from "../../selectors/assessment";
+import {questionCount, questions, outcomes }  from "../../selectors/assessment";
 
 const select = (state, props) => {
   return {
     settings             : state.settings,
     assessment           : state.assessment,
```

Comment:
@jbasdf Should we make that change in another branch when we remove immutable from the component?